changelog/v0.16.2/update-k8s-19.yaml

```diff
@@ -0,0 +1,5 @@
+changelog:
```

Comment:
I don't think we have a hard rule on when to do a minor vs patch on these 0.x repos, but this feels like more of a 0.17.0 I think?

Reply:
+1, let's mark this as a break as importing projects will probably have to make some changes to integrate this

Reply:
They will, especially when the `context` related changes are made